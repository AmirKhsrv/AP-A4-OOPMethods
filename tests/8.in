best_book
recommend_second_approach 92
show_author_info 1407
