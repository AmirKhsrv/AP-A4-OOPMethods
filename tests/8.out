id: 405
Title: Ellen Foster
Genre: Fiction
Author: Kaye Gibbons
Average Rating: 3.42
id: 211
Title: The Fall of Five
Genre: Young Adult
Author: Pittacus Lore
Reviews:
id: 76 Rating: 5 Likes: 0 Date: August 19 2016
id: 490 Rating: 4 Likes: 0 Date: January 26 2018
id: 1006 Rating: 1 Likes: 3 Date: January 25 2017
id: 1861 Rating: 4 Likes: 0 Date: February 9 2018
id: 2686 Rating: 0 Likes: 0 Date: September 11 2016
id: 2727 Rating: 5 Likes: 0 Date: November 26 2015
id: 3688 Rating: 2 Likes: 0 Date: September 9 2017
id: 3734 Rating: 1 Likes: 0 Date: April 15 2012
id: 4304 Rating: 3 Likes: 26 Date: September 21 2011
id: 4478 Rating: 4 Likes: 1 Date: March 27 2019
id: 4593 Rating: 5 Likes: 0 Date: January 16 2015
id: 4677 Rating: 4 Likes: 1 Date: September 1 2016
id: 4857 Rating: 5 Likes: 4 Date: September 19 2017
id: 5397 Rating: 5 Likes: 0 Date: October 10 2018
id: 6087 Rating: 1 Likes: 0 Date: June 9 2011
id: 6101 Rating: 4 Likes: 0 Date: March 9 2019
id: 7230 Rating: 1 Likes: 10 Date: December 29 2017
id: 7660 Rating: 5 Likes: 3 Date: April 24 2018
id: 8166 Rating: 5 Likes: 0 Date: January 6 2015
id: 8699 Rating: 1 Likes: 0 Date: July 25 2019
id: 9124 Rating: 0 Likes: 0 Date: February 9 2016
id: 9144 Rating: 3 Likes: 0 Date: December 15 2017
id: 9730 Rating: 1 Likes: 0 Date: June 9 2011
id: 9892 Rating: 2 Likes: 0 Date: May 27 2017
id: 10053 Rating: 5 Likes: 0 Date: October 12 2018
id: 10413 Rating: 5 Likes: 0 Date: December 14 2018
id: 10697 Rating: 0 Likes: 22 Date: April 26 2017
id: 10906 Rating: 0 Likes: 0 Date: October 8 2014
id: 10993 Rating: 1 Likes: 0 Date: October 20 2018
id: 11111 Rating: 2 Likes: 3 Date: February 28 2016
id: 11330 Rating: 0 Likes: 26 Date: September 29 2014
id: 11761 Rating: 5 Likes: 0 Date: April 24 2015
id: 11958 Rating: 3 Likes: 0 Date: October 23 2014
id: 12401 Rating: 1 Likes: 0 Date: August 13 2015
id: 12622 Rating: 0 Likes: 0 Date: November 12 2019
id: 13928 Rating: 2 Likes: 0 Date: November 29 2019
id: 14234 Rating: 2 Likes: 2 Date: March 15 2019
id: 14242 Rating: 1 Likes: 1 Date: April 4 2019
id: 14568 Rating: 0 Likes: 0 Date: February 27 2018
id: 16016 Rating: 3 Likes: 0 Date: December 20 2015
id: 16289 Rating: 4 Likes: 0 Date: August 14 2015
id: 16570 Rating: 4 Likes: 0 Date: October 13 2018
id: 16628 Rating: 1 Likes: 0 Date: January 10 2016
id: 16765 Rating: 4 Likes: 0 Date: October 27 2018
id: 1407
Name: Len Deighton
Year of Birth: 1981
Place of Birth: Sudan
Member Since: March 26 2008
Genres: Historical, Thriller
Books:
